tests/unit/operators/test_ai_catalog.py

```diff
@@ -29,3 +30,27 @@ def test_operator_upload_dataset(mocker):
 
     assert dataset_id == "dataset-id"
     upload_dataset_mock.assert_called_with("/path/to/local/file")
+
+
+def test_operator_update_dataset_from_file(mocker):
```

Comment:
I'd suggest to parametrize this test to also test the case where operator is initiliazed with  `UpdateDatasetFromFileOperator(dataset_id=<dataset_id>)`.